tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Delete.feature

```diff
@@ -0,0 +1,61 @@
+﻿Feature: Delete an already existing order
+    As an Buyer user
+    I want to be able to delete a given order
+    So that I can... delete my order?
+
+Background:
+    Given Orders exist
+        | OrderId    | Description   | OrganisationId                       | OrganisationName | OrganisationOdsCode | OrganisationContactEmail | SupplierAddressPostcode | SupplierContactEmail    |
+        | C000014-01 | A Description | 4af62b99-638c-4247-875e-965239cd0c48 | Hampshire CC     | 432432              | Fred.robinson@email.com  | LS 1 3AP                | Fred.robinson@email.com |
+        And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+Scenario: 1. Delete an existing order
+    Given the user creates a request to delete the order with ID 'C000014-01'
+    When the user sends the delete order request
+    Then a response with status code 204 is returned
+    And the expected order is deleted
+    And the order with orderId C000014-01 is updated in the database with data

```

Comment:
Would it be better to move this last assertion to another scenario as it looks like a secondary piece of behaviour? Hopefully makes the test easier to read & understand as you are testing one specific piece of change.

Reply:
roger roger, good call